src/modules/fundingManager/oracle/interfaces/IOraclePrice_v1.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+/**
+* @title   Oracle Price Interface
+* @notice  Interface for oracle price feed calculations
+* @dev     Designed to facilitate various oracle price implementations
+* @custom:security-contact security@inverter.network
+* @author  Zealynx Security
+*/
+interface IOraclePrice_v1 {
```

Comment:
When we talked last time you mentioned to keep an oracle interface because eventually this will/might come from an actual oracle, so I thought of leaving it for that purpose.

Reply:
formatted